everyvoice/cli.py

```diff
@@ -268,17 +271,50 @@ class ModelTypes(str, Enum):
     short_help="Export your EveryVoice models",
 )
 
-app.command(
-    short_help="Segment a long audio file",
-    name="segment",
+# Add the segment commands
+segment_group = typer.Typer(
+    pretty_exceptions_show_locals=False,
+    context_settings={"help_option_names": ["-h", "--help"]},
+    rich_markup_mode="markdown",
+    cls=TyperGroupOrderAsDeclared,
+    help="""
+    # Segment Help
+
+        - **align** --- This command will align a long audio file with some text.
```

Comment:
What's missing here is clarity on the granularity of segmentation. Are you segmenting to words, sentences, phonological utterances? On this line, a very brief reference to the granularity level would be good.